lib/tinybucket/api/branches_api.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Tinybucket
+  module Api
+    # Branches Api client
+    #
+    # @!attribute [rw] repo_owner
+    #   @return [String] repository owner name.
+    # @!attribute [rw] repo_slug
+    #   @return [String] repository slug. (about {https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D
```

Comment:
Currently, this yard comment shows a raw URL in generated document. 😲 

You should add ` repo_slug})` after this line to create the link in yard document. 📘